.github/workflows/go.yml

```
@@ -0,0 +1,33 @@
+name: Go
+
+on:
+  push:
+    branches: [ "**" ]
+  pull_request:
+    branches: [ master ]
```

Looks like we have main as default branch 👍 
```suggestion
    branches: [ main ]
```